python/data/ArtusConfigs/Includes/settingsKappa.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import Artus.Utility.logger as logger
+log = logging.getLogger(__name__)
+
+import re
+import json
+import copy
+import Artus.Utility.jsonTools as jsonTools
+import Kappa.Skimming.datasetsHelperTwopz as datasetsHelperTwopz
+import importlib
+
+def build_config(nickname):
+  config = jsonTools.JsonDict()
+  datasetsHelper = datasetsHelperTwopz.datasetsHelperTwopz("Kappa/Skimming/data/datasets.json")
+  
+  
+  ## fill config:
+  # includes
+  includes = [
+    ]
+  for include_file in includes:
+    analysis_config_module = importlib.import_module(include_file)
+    config += analysis_config_module.build_config(nickname)
+  
+  # explicit configuration
+  config["GenParticles"] = "" if re.search("(?<!PFembedded).Run201", nickname) else "genParticles"
+  config["GenTaus"] = "" if re.search("(?<!PFembedded).Run201", nickname) else "genTaus"
+  config["GenTauJets"] = "" if re.search("(?<!PFembedded).Run201|8TeV", nickname) else "tauGenJets"
+  config["GenMet"] = "" if re.search("(?<!PFembedded).Run201", nickname) else "genmetTrue"
+  config["GenJets"] = ""
+  config["Electrons"] = "electrons"
+  config["ElectronMetadata"] = "electronMetadata"
+  config["Muons"] = "muons"
+  config["Taus"] = "taus"
+  config["TauMetadata"] = "taus"
+  
+  if re.search("8TeV|13TeV.*_AODSIM", nickname):     config["TaggedJets"] = "AK5PFTaggedJets"
```

Comment:
I guess we can drop 8 TeV and AOD support?

Reply:
done